### PR TITLE
Add badges to readme

### LINKED ISSUE #3: Add badges
Add badges similar to https://github.com/arshadkazmi42/flip-remote

### PR DESCRIPTION
**Pull request by Easybuoy**
#### What does this PR do?
Add badges to readme file

#### Description of Task to be completed?
- Edit Readme file, and add `repo size`, `license`, `downloads`, `npm`, `contributions`, `commits` badges.

#### How should this be manually tested?
- Clone the branch <code>add-badges-to-readme</code>
- Check the Readme file to see added badges.

### Screenshots (if appropriate)
Screenshot 2019-10-01 at 6 57 53 PM

Closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # process-name
 
 [![Build](https://img.shields.io/travis/com/arshadkazmi42/process-name.svg)](https://travis-ci.com/arshadkazmi42/process-name/)
+[![Github Repo Size](https://img.shields.io/github/repo-size/arshadkazmi42/flip-remote.svg)](https://github.com/arshadkazmi42/process-name)
```
Comment (arshadkazmi42) [inline README.md]: Change `flip-remote.svg` to `process-name.svg`

File: README.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+[![LICENSE](https://img.shields.io/npm/l/flip-remote.svg)](https://github.com/arshadkazmi42/process-name/blob/master/LICENSE)
```
Comment (arshadkazmi42) [inline README.md]: Change `flip-remote.svg` to `process-name.svg`

File: README.md [hunk 3]
```diff
@@ -1,6 +1,12 @@
...
+[![NPM Downloads](https://img.shields.io/npm/dt/flip-remote.svg)](https://www.npmjs.com/package/process-name)
```
Comment (arshadkazmi42) [inline README.md]: Change `flip-remote.svg` to `process-name.svg`

File: README.md [hunk 4]
```diff
@@ -1,6 +1,12 @@
...
+[![NPM Version](https://img.shields.io/npm/v/flip-remote.svg)](https://www.npmjs.com/package/process-name)
```
Comment (arshadkazmi42) [inline README.md]: Change `flip-remote.svg` to `process-name.svg`

File: README.md [hunk 5]
```diff
@@ -1,6 +1,12 @@
...
+[![Contributors](https://img.shields.io/github/contributors/arshadkazmi42/flip-remote.svg)](https://github.com/arshadkazmi42/process-name/graphs/contributors)
```
Comment (arshadkazmi42) [inline README.md]: Change `flip-remote.svg` to `process-name.svg`

File: README.md [hunk 6]
```diff
@@ -1,6 +1,12 @@
...
+[![Commit](https://img.shields.io/github/last-commit/arshadkazmi42/flip-remote.svg)](https://github.com/arshadkazmi42/process-name/commits/master)
```
Comment (arshadkazmi42) [inline README.md]: Change `flip-remote.svg` to `process-name.svg`

**Comment by Easybuoy** [comment]
> Looks awesome 💯
> You have missed flip-remote text in couple for places

I have made requested changes

**Comment by arshadkazmi42** [comment]
Thank  you for working on this

**Pull request merged**